### PR TITLE
Linking elements in the settings page with external urls r=wilsonpage

### PR DESCRIPTION
**Pull request by arcturus**
@wilsonpage added the links in the config file to make it easier to change ... if we have to ;)

### CODE REVIEW

File: lib/views/menu-scene.js [hunk 1]
```diff
@@ -69,6 +78,16 @@ class ListScene extends React.Component {
   onClosePress() {
     this.props.navigator.pop();
   }
+  onItemPress(id) {
+    const url = settings.links[id];
+    if (!url) {
+      return;
+    }
```
Comment (wilsonpage) [inline lib/views/menu-scene.js]: _cough_

File: lib/views/menu-scene.js [hunk 2]
```diff
@@ -69,6 +78,16 @@ class ListScene extends React.Component {
...
+
+    Linking.openURL(url).catch(err => {
+      console.info(`Could not open url ${url} `, err);
+    });
```
Comment (wilsonpage) [inline lib/views/menu-scene.js:88]: Under what circumstance would this fail? I imagine only for malformed URLs. In which case, worth it?

**Comment by wilsonpage** [comment]
Arrrr pluuussssss ;)

File: lib/views/menu-scene.js [hunk 3]
```diff
@@ -69,6 +78,16 @@ class ListScene extends React.Component {
   onClosePress() {
     this.props.navigator.pop();
   }
```
Comment (wilsonpage) [inline lib/views/menu-scene.js:80]: NIT: line-break between blocks.

**Comment by wilsonpage** [comment]
After nits :)

File: lib/views/menu-scene.js (hunk 1 above)
Comment (arcturus) [inline lib/views/menu-scene.js]: I did it in purpose, can i keep it? It's so nice and stylish

**Comment by arcturus** [comment]
Ok, Ok, it's really hard for me to write 

``` javascript
if (!url) { return; }
```

**Pull request merged**